tests/Unit/Application/Asset/HandleAssetProcessingRetryExhaustionServiceTest.php

```
@@ -27,15 +27,15 @@ final class HandleAssetProcessingRetryExhaustionServiceTest extends TestCase
     private const UNKNOWN_ASSET_ID = '123e4567-e89b-42d3-a456-426614174000';
 
     private AssetRepositoryInterface&MockObject $assets;
-    private AssetTerminalStatusCacheInterface&MockObject $assetTerminalStatusCache;
+    private AssetStatusCacheInterface&MockObject $assetTerminalStatusCache;
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $assetTerminalStatusCache. Keep variable name length under 20. (undefined)

(LongVariable)